src/cs-graphics/internal/gltfmodel.cpp

```diff
@@ -581,8 +581,9 @@ Buffer getOrCreateBufferObject(std::map<int, Buffer>& bufferMap, tinygltf::Model
   auto it = bufferMap.find(bufferViewIndex);
   if (it != bufferMap.end()) {
     if (it->second.target != target) {
-      std::cerr << "WARNING: getOrCreateBufferObject target is different"
-                << " from Buffer.target for " << bufferViewIndex << '\n';
+      spdlog::error(
```

Comment:
Will this lead to an application crash? The original was only printing a warning.
```suggestion
      spdlog::warn(
```